src/main/java/be/xplore/pricescraper/utils/ProductMatcher.java

```diff
@@ -0,0 +1,74 @@
+package be.xplore.pricescraper.utils;
+
+import org.apache.commons.text.similarity.LevenshteinDistance;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductMatcher {
+
+    private final int MATCH_THRESHOLD;
+
+    public ProductMatcher(@Value("${productmatcher.ingredients.threshold}") int threshold) {
```

Comment:
`@Value` won't work if this class isn't a component. Property could be injected in a bean that instantiates this class in the future though.